build.sbt

```diff
@@ -1120,18 +1125,19 @@ lazy val `interpreter-dsl` = (project in file("lib/scala/interpreter-dsl"))
 // === Sub-Projects ===========================================================
 // ============================================================================
 
-val truffleRunOptionsNoAssert = if (java.lang.Boolean.getBoolean("bench.compileOnly")) {
-  Seq(
-    "-Dpolyglot.engine.IterativePartialEscape=true",
-    "-Dpolyglot.engine.BackgroundCompilation=false",
-    "-Dbench.compileOnly=true"
-  )
-} else {
-  Seq(
-    "-Dpolyglot.engine.IterativePartialEscape=true",
-    "-Dpolyglot.engine.BackgroundCompilation=false"
-  )
-}
+val truffleRunOptionsNoAssert =
+  if (java.lang.Boolean.getBoolean("bench.compileOnly")) {
+    Seq(
+      "-Dpolyglot.engine.IterativePartialEscape=true",
+      "-Dpolyglot.engine.BackgroundCompilation=false",
+      "-Dbench.compileOnly=true"
+    )
+  } else {
+    Seq(
+      "-Dpolyglot.engine.IterativePartialEscape=true",
+      "-Dpolyglot.engine.BackgroundCompilation=false"
+    )
+  }
```

Comment:
I did not touch these, just ran `scalafmt` as we are supposed to. Seems like it was not ran for a while. Please choose to ignore whitespace changes.